blueocean-dashboard/src/main/js/components/RunDetailsHeader.jsx

```diff
@@ -77,29 +77,30 @@ class RunDetailsHeader extends Component {
                 <a className="path-link" onClick={ onNameClick }>
                     <ExpandablePath path={ fullDisplayName } hideFirst className="dark-theme" iconSize={ 20 } />
                 </a>
-                <span>#{ run.id }</span>
+                <span>&nbsp;#{ run.id }</span>
             </h1>
         );
 
         const branchSourceDetails = (
-            <div>
+            <div className="u-label-value" title={branchLabel + ': ' + displayName}>
                 <label>{ branchLabel }:</label>
                 <span>{ displayName }</span>
             </div>
         );
 
-        const commitSourceDetails = run.commitId && (
-                <div>
-                    <label>{ commitLabel }:</label>
-                    <span className="commit">
-                         { run.commitId.substring(0, 7) }
-                    </span>
-                </div>
-            );
+        const commitIdString = run.commitId || 'N/A';
+        const commitSourceDetails = (
+            <div className="u-label-value" title={commitLabel + ': ' + commitIdString}>
+                <label>{ commitLabel }:</label>
+                <span className="commit">
+                     { commitIdString.substring(0, 7) }
```

Comment:
I know this isn't new, but it makes me sad... format should be specified by the scm plugin somehow